### ISSUE TITLE
Multisite Network Activation

### ISSUE THREAD

**Issue by fym** [palasthotel/grid-wordpress#90]
It seems that if you activate grid network wide some or all of each subsites' needed grid database tables won't be created. The plugin itself is still active, all the menu items are available, but menu pages throw a database notice (e.g. settings page)

**Comment by JanaME** [comment]
Yes, I’m running into the same problem here @EdwardBock  :( The workaround is of course activating grid for each site instead of activating it sidewide, but it’s a bit annoying when working with more than 10 sites…
